sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala

```diff
@@ -34,14 +34,16 @@ import org.apache.spark.util.kvstore.{KVIndex, KVStore}
  * Provides a view of a KVStore with methods that make it easy to query SQL-specific state. There's
  * no state kept in this class, so it's ok to have multiple instances of it in an application.
  */
-class SQLAppStatusStore(
-    store: KVStore,
-    val listener: Option[SQLAppStatusListener] = None) {
+class SQLAppStatusStore(store: KVStore, val listener: Option[SQLAppStatusListener] = None) {
```

Comment:
Nothing is changing here? If not just leave it as it was.

Reply:
`scalafmt` corrected the entire file, corrected the PR to format only modified code